.travis.yml

```diff
@@ -1,17 +1,28 @@
+dist: xenial
 language: python
 git:
   depth: 50 # Need old commits for determining version of untagged builds
-env: DJANGO=1.11.9
+env: DJANGO=1.11
 matrix:
   include:
   - python: 2.7
-  - python: 3.4
-  - python: 2.7 # Old Middleware classes
-    env: DJANGO=1.9
-  - python: 3.4 # Django 2 support
-    env: DJANGO==2.0
+  - python: 3.5
+  - python: 3.6
+  - python: 3.7
+  - python: 3.5 
+    env: DJANGO=2.1
+  - python: 3.6
+    env: DJANGO=2.1
+  - python: 3.7
+    env: DJANGO=2.1
+  - python: 3.5 
+    env: DJANGO=2.2
+  - python: 3.6
+    env: DJANGO=2.2
+  - python: 3.7
+    env: DJANGO=2.2
```

Comment:
Do we really need to test Python 3.5/6/7 against *both* Django 2.1 and 2.2? Else, I'd stick to test those Python versions against only one (perhaps 2.2, as it's the newer ;) )

Reply:
I believe that it's basically a good idea, since even a minor version change can lead to a breakage when we are talking about such a huge project as `Django`.

Reply:
I can change this however you guys think it should be.
Having said that... I don’t see a real downside to running this under multiple Django/Python  version (as long as they are supported).

Reply:
@carlosalberto updated (now only testing python 3.x version with Django 2.2)
I would love to get this merged and published to pypi